pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -628,9 +628,9 @@ message Image {
     // ID of the image.
     optional string id = 1;
     // Other names by which this image is known.
-    repeated string repo_tags = 2;
+    repeated string names = 2;
```

Comment:
@feiskyer We can comment and require the `Name` to have format like `image_name:tag` as well. `RepoTags` just sounds very docker specific.  Same as `RepoDigests`.

@yujuhong I don't see why the `Names` in node status should contain digest at first place? Shouldn't we add a field about the digest / hash of the image?